jestSetup.js

```diff
@@ -0,0 +1,3 @@
+jest.mock('./lib/storage/providers/SQLiteStorage', () =>
+  require('./lib/storage/providers/__mocks__/SQLiteStorage')
+);
```

Comment:
I'm not sure why this was added - to active a manual mock from the `__mocks__` folder you just call

```
jest.mock('./lib/storage/providers/SQLiteStorage')
```

In the test file that is supposed to use a mocked version